sda-auth/dev-server/docker-compose.yml

```diff
@@ -19,6 +19,11 @@ services:
       - USERINFO_ROUTE=/userinfo
     ports:
       - 9090:9090
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9090/.well-known/openid-configuration"]
```

Comment:
Is this a real URL that will actually return something?

Reply:
yes, this is the `/.well-known/openid-configuration` endpoint of the`mockoidc`